scripts/ngless-cwl/cwl/ngless-unique.cwl

```diff
@@ -6,40 +6,40 @@
 cwlVersion: "cwl:v1.0"
```

Comment:
If you've hand edited a CWL description made by `argparse2tool` you might want to remove the lines about generating them again (3 & 4) and make explicit that manual changes have been made (line 2)

How about
```
# This tool description was originally generated automatically by argparse2tool ver. 0.4.3-2
# but has since been modified by hand
```
?